package.json

```
@@ -31,8 +32,22 @@
   },
   "devDependencies": {
     "eslint": "^8.56.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^15.2.0",
     "live-server": "^1.2.2",
     "prettier": "^3.1.1",
     "sass": "^1.69.7"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "*.html": [
+      "prettier --write"
+    ],
+    "*.scss": [
+      "prettier --write"
+    ]
```

```suggestion
    "*.{html,scss}": [
      "prettier --write"
    ]
```

What do you think about combining these into a single block?